src/node/routes/vscode.ts

```diff
@@ -37,19 +37,15 @@ export const createVSServerRouter = async (args: DefaultedArgs): Promise<VSServe
 
   // Signal processes that we got launched as CLI
   process.env["VSCODE_CLI"] = "1"
+  // Seems to be necessary to load modules properly.
+  process.env["VSCODE_DEV"] = "1"
```

Comment:
Is this dev as in development?  I wonder if this will cause issues in the production build.

Reply:
I suspect this is yet to be fixed upstream since their server run command [assumes a development env](https://github.com/microsoft/vscode/blob/8b8e1d1dfaf6fa6e42c4ca2743c40a012c4ac455/resources/server/web.sh#L21), and their [prod command](https://github.com/microsoft/vscode/blob/main/resources/server/bin/server.sh) seems to error without this arg.